### DISCUSSION TITLE
Object detection without AVX/AVX2 cpu flags?

### DISCUSSION THREAD

**Discussion by MozPri** [immich-app/immich#876] [Q&A] [answered]
I noticed objects are getting detected on the Server Stats page even though my CPU does not support AVX/AVX2 instruction set (Intel Q6600).
I thought that AVX CPU flags were necessary for object detection. Is this correct?

![image](https://user-images.githubusercontent.com/8180464/198272152-a6c900ad-2a29-41fb-8811-51de4e6bf56f.png)

Although in mobile app > search only places are detected. 
For "Things" there is only a message "No Objects Info Available"

**Comment by bo0tzz** [comment, marked as answer]
I took a look at the code - it seems like `objects` in the stats isn't actually the count of AI-detected objects, but rather the amount of files on disk. That also includes thumbnails etc. A little confusing, IMO. @alextran1502 thoughts?

**Reply by alextran1502** [reply]
Yes the wording is confusing, let's change it

**Comment by bertmelis** [comment]
If you compile tensorflow from source, you can have object detection: #300
